sem/HW/forms.py

```diff
@@ -0,0 +1,17 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Email, Length, EqualTo
+
+
+class RegistrationForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    usersurname = StringField('Usersurname', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Можно использовать поле `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```